lib/liquid_voting/release.ex

```diff
@@ -11,6 +13,26 @@ defmodule LiquidVoting.Release do
     {:ok, _, _} = Ecto.Migrator.with_repo(repo, &Ecto.Migrator.run(&1, :down, to: version))
   end
 
+  @doc """
+  Deletes votes, participants and delegations for the smoke tests organization_id.
+  For use before and after running smoke tests.
+  """
+  def teardown() do
+    organization_id = "bc7eeccb-5e10-4004-8bfb-7fc68536bbd7"
```

Comment:
it's good practice to define constants outside of functions, to keep them nice and visible when browsing the source. Here you could extract this and define it as a module attribute like this:

```
defmodule LiquidVoting.Release do
  @app :liquid_voting
  @test_organization_id "bc7eeccb-5e10-4004-8bfb-7fc68536bbd7"
```

(prepending _test__ makes it more explicit too)